tests/clients/test_job.py

```diff
@@ -84,7 +99,47 @@
 
 JOB_SUMMARY = JobSummary(id=JOB_ID, metadata=JOB_METADATA)
 JOB_SUMMARY_BODY = {"jobId": str(JOB_ID), "meta": JOB_METADATA_BODY}
-
+INSTANCE_SIZE = InstanceSize(milli_cpus=MILLI_CPUS, memory_mb=MEMORY_MB)
+INSTANCE_SIZE_BODY = {"milliCpus": str(MILLI_CPUS), "memoryMb": str(MEMORY_MB)}
+JOB_PARAMETERS = JobParameter(
+    name="job name",
+    param_type=ParameterType.TEXT,
+    default="default parameter",
+    required=True,
+)
+JOB_PARAMETERS_BODY = {
+    "name": JOB_PARAMETERS.name,
+    "type": "text",
+    "default": JOB_PARAMETERS.default,
+    "required": str(True),
+}
+JOB_COMMAND = JobCommand(name="command name", parameters=[JOB_PARAMETERS])
```

Comment:
Unfortunately, the command 'name' is the actual command that gets run (it's a very poor choice of name that I tried, unsuccessfully, to change at the time). For clarity, could we do something like:
```suggestion
JOB_COMMAND = JobCommand(name="python myscript.py", parameters=[JOB_PARAMETERS])
```